[bug] Incorrect bindings generation

Hello :)

I'm having an issue with how componentize.py generate bindings.

I have the following dataflow.wit file

```
package dataflow:processor;

world data-flow-processor {

    enum frame-type {
        CAN,
        LIN,
        UNKNOWN
    }

    record frame {
        frame-type: frame-type,
        id: s32,
        channel-no: s32,
        source-id: string,
        source-name: string,
        data: list<u8>,
        data-length: s32,
        dlc: s32,
        flag: s32,
        time: f64
    }

    import output: func(frame: frame) -> ();
}
```

If I generate bindings with `componentize-py -d dataflow.wit bindings .` the following `__init__.py` is generated:
```python
from typing import TypeVar, Generic, Union, Optional, Protocol, Tuple, List, Any, Self
from types import TracebackType
from enum import Flag, Enum, auto
from dataclasses import dataclass
from abc import abstractmethod
import weakref

from .types import Result, Ok, Err, Some


def output(frame: Frame) -> None:
    raise NotImplementedError


class FrameType(Enum):
    CAN = 0
    LIN = 1
    UNKNOWN = 2

@dataclass
class Frame:
    frame_type: FrameType
    id: int
    channel_no: int
    source_id: str
    source_name: str
    data: bytes
    data_length: int
    dlc: int
    flag: int
    time: float

class DataFlowProcessor(Protocol):
    pass
```

As you can see the `output` function is declared before the `Frame` class later producing the following error when I try to create my component:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\examples\python\basic\venv\Scripts\componentize-py.exe\__main__.py", line 7, in <module>
    sys.exit(script())
             ~~~~~~^^
AssertionError: Traceback (most recent call last):
  File "/0/basic.py", line 1, in <module>
    from data_flow_processor import Frame, FrameType
  File "/world/data_flow_processor/__init__.py", line 12, in <module>
    def output(frame: Frame) -> None:
                      ^^^^^
NameError: name 'Frame' is not defined


Caused by:
    NameError: name 'Frame' is not defined
```

Is this a bug or does anyone know any workaround? Thanks in advance :D

--- a/src/summary.rs
+++ b/src/summary.rs
@@ -1986,8 +1986,8 @@ from ..types import Result, Ok, Err, Some
                 "{docs}{python_imports}
 from .types import Result, Ok, Err, Some
 {imports}
-{function_imports}
 {type_exports}
+{function_imports}
 {protocol}
 "
             )?;

--- a/src/test/tests.rs
+++ b/src/test/tests.rs
@@ -57,6 +57,13 @@ pub struct ThingList(Vec<u8>);
 pub struct ThingString(String);
 pub struct MyFloat(f64);
 
+#[async_trait]
+impl TestsImports for Ctx {
+    async fn output(&mut self, _: Frame) -> Result<()> {
+        unreachable!()
+    }
+}
+
 macro_rules! load_guest_code {
     ($($input_string:expr),*) => {
         &[

--- a/src/test/wit/tests.wit
+++ b/src/test/wit/tests.wit
@@ -179,4 +179,10 @@ world tests {
   export add: func(a: borrow<float>, b: borrow<float>) -> own<float>;
 
   export read-file: func(path: string) -> result<list<u8>, string>;
+
+  record frame {
+    id: s32,
+  }
+
+  import output: func(frame: frame);
 }

